### PR TITLE
Adds some stuff for the librarian to do 

### PR DESCRIPTION
**Pull request by harmonyn**
* The library computer can now upload scanned books to the newscaster, opening up a whole new <s>wood</s> world. 
* The library computer can now print corporate posters as well as Bibles. Removes corporate posters crate from cargo, as it is now obsolete (and literally no one bought it anyway).

:cl: bgobandit
add: The library computer can now upload scanned books to the newscaster. Remember, seditious or unsavory news channels should receive a Nanotrasen D-Notice!
add: The library computer can now print corporate posters as well as Bibles.
del: Cargo no longer offers a corporate poster crate. Nobody ever bought it anyway.
/:cl:

### CODE REVIEW

**Comment by Ninja137** [comment]
But can the library be emagged to print contraband posters?

**Comment by bgobandit** [comment]
I guess I could add that functionality but contraband posters are already strewn like candy all over the goddamn station

**Comment by coiax** [comment]
To be honest, mapmakers should use a different poster type for corporate and contraband posters.

**Comment by Anonmare** [comment]
I think contraband posters should be a multitool thing. Not everyone has an emag for one but multitools are significantly easier to get.

**Comment by ExcessiveUseOfCobblestone** [comment]
yes please move away from the emag

File: code/modules/library/lib_machines.dm [hunk 1]
```diff
@@ -453,7 +470,27 @@ var/global/list/datum/cachedbook/cachedbooks // List of our cached book datums
...
+	if(href_list["printbible"])
+		if(!bibledelay)
+			var/obj/item/weapon/storage/book/bible/B = new /obj/item/weapon/storage/book/bible(src.loc)
+			if(SSreligion.Bible_icon_state && SSreligion.Bible_item_state)
+				B.icon_state = SSreligion.Bible_icon_state
+				B.item_state = SSreligion.Bible_item_state
+				B.name = SSreligion.Bible_name
+				B.deity_name = SSreligion.Bible_deity_name
+			bibledelay = 1
+			spawn(PRINTER_COOLDOWN)
```
Comment (KorPhaeron) [inline code/modules/library/lib_machines.dm]: We do addtimer now, not spawns

Comment (bgobandit) [inline code/modules/library/lib_machines.dm]: syntax please (this was there from the start)

**Comment by optimumtact** [comment]
lovely

**Comment by AnturK** [comment]
This was waiting for bgo to fix the alert()'s

**Comment by optimumtact** [comment]
Then why didn't it say so on the PR

**Pull request merged**